var/spack/repos/builtin/packages/dftd4/package.py

```diff
@@ -44,8 +44,16 @@ def meson_args(self):
             lapack = "auto"
 
         return [
-            "--wrap-mode=nodownload",
             "-Dlapack={0}".format(lapack),
             "-Dopenmp={0}".format(str("+openmp" in self.spec).lower()),
             "-Dpython={0}".format(str("+python" in self.spec).lower()),
         ]
+
+    def setup_run_environment(self, env):
```

Comment:
You can remove this and replace:
```python
depends_on("python@3.6:", when="+python")
```
with:
```python
extends("python", when="+python")
```
and it will have the same effect.